config/src/main/java/org/axonframework/config/AxonIQConsoleModule.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2010-2023. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.config;
+
+import org.axonframework.util.MavenArtifactVersionResolver;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.lang.invoke.MethodHandles;
+import java.util.Objects;
+import javax.annotation.Nullable;
+
+/**
+ * Module related to AxonIQ console, for now it's just logging information about AxonIQ console when it's not suppressed
+ * and the console-framework-client is not available.
+ *
+ * @author Gerard Klijs
+ * @since 4.9.0
+ */
+public class AxonIQConsoleModule implements ModuleConfiguration {
+
+    private static final Logger logger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+    private static final String DISABLE_CONSOLE_MESSAGE_SYSTEM_PROPERTY = "disable-console-message";
```

Comment:
I would make it a little more specific;
```suggestion
    private static final String DISABLE_CONSOLE_MESSAGE_SYSTEM_PROPERTY = "disable-axoniq-console-message";
```